indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -425,6 +426,22 @@
   [context concept]
   (meta-db/get-associations-for-tool context concept))
 
+(defn create-association-map-keys
+  "Creates the association map keys that lists out all of the different association types.
+  For example, this function changes the named concept type from variable-association to a 
+  key of :variable-associations."
+  [key-str]
+  {key-str (keyword (str key-str "s")) })
```

Comment:
I didn't think of it.  Based on Leo's comment below though, I will be getting rid of this above function.  It just depends on if I get rid of it now or in January through another ticket.  The inflections library does work.  OK, I will add it for now, and then depending on what Leo and I agree on, it will either go away now or in January.